.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,25 +7,37 @@ assignees: ''
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
-
-**Steps To Reproduce**
-Steps to reproduce the behavior:
-1.
-2.
-...
+<!--
+This repository tracks issues related to the implementation of the Swift
+compiler, standard library, runtime, and tools that provide IDE support for
+Swift (e.g. code completion). If the bug relates to the implementation of a
+proprietary (closed-source) Apple framework such as UIKit, SwiftUI, Combine,
+etc., please report it to https://feedbackassistant.apple.com instead.
+-->
+
+**Description**
+<!-- Describe clearly and concisely what the bug is. -->
+
+**Steps to reproduce**
+<!--
+Explain how to reproduce the problem (in steps if seen fit) and include a
+reproducible test case, preferably reduced.
+-->
 
 **Expected behavior**
-A clear and concise description of what you expected to happen.
-
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
-
-**Environment (please fill out the following information)**
- - OS: [e.g. macOS 11.0]
- - Xcode Version/Tag/Branch:
-
-
-**Additional context**
-Add any other context about the problem here.
+<!-- Describe what you expected to happen. -->
+
+<!-- If deemed helpful, add screenshots that showcase the problem. -->
+<!-- **Screenshots** -->
+
+<!--
+Include information about the Swift compiler version and, if applicable, the
+Xcode version you are observing the problem in and the deployment target.
+-->
+**Environment**
+- Swift compiler version info <!-- replace with the output of 'swiftc -version' -->
```

Comment:
`:` in both cases?

Reply:
I deliberately omitted a colon as a subtle indication to replace the text. `swiftc -version` already prints "Swift version", so it would sound tautological.

Reply:
I see, sounds good!

Reply:
Just to float an alternative: GitHub has [this issue form thing](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) that avoids the ambiguity of what text should be replaced in the template, although it's perhaps too much of an overkill.

Reply:
Yeah, I figured I would give it a go once it fully releases.